apps/cookbook/src/app/examples/chart-example/examples/chart-example-monthly-overview.component.ts

```diff
@@ -0,0 +1,142 @@
+import { Component } from '@angular/core';
+import { Options, SeriesClickCallbackFunction, SeriesClickEventObject } from 'highcharts';
+
+import { ModalController, DesignTokenHelper } from '@kirbydesign/designsystem';
+import { Example } from '../../example.interface';
+
+const getColor = DesignTokenHelper.getColor;
+function colorPoints(selectedIdx) {
+  return function() {
+    var series = this.series;
+    for (var i = 0, ie = series.length; i < ie; ++i) {
+      var points = series[i].data;
+      for (var j = 0, je = points.length; j < je; ++j) {
+        if (points[j].graphic) {
+          if (j === selectedIdx) {
+            points[j].graphic.element.style.fill = getColor('primary').value;
+            points[j].graphic.element.style.stroke = getColor('primary').value;
+          } else {
+            points[j].graphic.element.style.stroke = getColor('secondary').value;
+          }
+        }
+      }
+    }
+  };
+}
+
+const config = {
+  selector: 'cookbook-chart-example-monthly-overview',
+  template: `
+  <kirby-card>
+  <kirby-card-header [title]="'Column'"></kirby-card-header>
+  <kirby-chart [height]="height" [categories]="categories" [type]="'column'" [data]="adjustedMonthlyExpenseData" [options]="monthlyOverviewOptions"> </kirby-chart>
+  </kirby-card>
+  `,
+  codeSnippet: `
+  private monthlyOverviewClick: SeriesClickCallbackFunction = (ev: SeriesClickEventObject) => {
+    this.modalController.showAlert({
+      title: 'Clicked chart',
+      message: 'You clicked on column: ' + ev.point.category,
+      okBtnText: 'Ok',
+    });
+
+    this.selectedIdx = this.categories.indexOf(ev.point.category);
+    this.monthlyOverviewOptions.plotOptions.series.animation = false;
+    this.monthlyOverviewOptions.chart.events.load = colorPoints(this.selectedIdx);
+    this.monthlyOverviewOptions.chart.events.redraw = colorPoints(this.selectedIdx);
+    this.monthlyOverviewOptions = { ...this.monthlyOverviewOptions };
+  };
+  monthlyOverviewOptions: Options = {
+    chart: { events: {} },
+    plotOptions: {
+      series: {},
+      column: {
```

Comment:
Why all this boilerplate? Should be removed

Reply:
Because in the eventhandler we try to access nested values of these.